cmd/server/cadence/server.go

```diff
@@ -107,7 +107,7 @@ func (s *server) startService() common.Daemon {
 	var err error
 
 	params := service.BootstrapParams{}
-	params.Name = "cadence-" + s.name
```

Comment:
I think this will not work. We intended to put "cadence-" as prefix, so that in the command line people just use "frontend,history,matching". It's already in a lot of README and scripts, this will be a big change but no necessary.